org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/DocumentProvider.xtend

```diff
@@ -85,7 +85,7 @@ class DocumentProvider {
 	private def void commit(File file) {
 		val git = gitProvider.git
 		git.stage(file)
-		git.commit.setMessage("bla").call
+		git.commit.setMessage("modify " + file.name).call
```

Comment:
There's also this #36 now as a potential alternative, but we can drop that PR if this here is sufficient to fulfill the requirements (if there are any…).